bus/bus.c

```diff
@@ -163,8 +170,10 @@ static struct bus_type gip_bus_type = {
 	.name = "xone-gip",
 	.match = gip_bus_match,
 	.probe = gip_bus_probe,
-#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 15, 0)
```

Comment:
With the above change only check for kernel below 5.15.0 is needed, just as it already is.

Reply:
I see what you mean, that's nice and clean! Will force push in a moment.